### ISSUE TITLE
<08/09/23><NYU><Android><Talkback><Purple Rides><When user cancelling the ride call driver pop-up is not visible> <Parent_Id-1980>

### ISSUE THREAD

**Issue by krupal-14** [nammayatri/nammayatri#3088]
#1980 
[When user cancelling the ride call driver option is not visible](https://drive.google.com/file/d/1r3d4lhLxuR0aeMKuFm-_m_p6aR8rhd6s/view?usp=sharing)

Steps To reproduce
1.launch the apk.
2.Login with valid credentials.
3.Booka ride.
4.cancel a ride and observe

Expected Result
Call driver pop-up should display.

Actual result
Call driver pop-up not dispalying.

**Comment by MercyQueen** [comment]
fixed.
